google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/AbstractDebugTarget.java

```diff
@@ -0,0 +1,9 @@
+package com.google.gct.idea.debugger.ui;
+
+/**
+ * Parent class to serve as a base for actual debug targets or specific child classes used e.g. for
+ * error handling
+ */
+public abstract class AbstractDebugTarget {
```

Comment:
The idea was here to use a consistent base type for the combobox used to display debug targets. Unfortunately Java supports generic JComboBox only from JDK7 and up, so we can happily mix different types as items without the compiler complaining.